format/Flight.proto

```diff
@@ -183,6 +183,15 @@ message Action {
   bytes body = 2;
 }
 
+/*
+ * The request of the RenewFlightEndpoint action.
+ *
+ * The request should be stored in Action.body.
+ */
+message RenewFlightEndpointRequest {
```

Comment:
Hmm, at this point should we just have request/result messages for all 3 actions?

Reply:
Ah, I saw the ML message just now, ignore me.

Reply:
Ah, sorry. I'm still changing now. I'll add `CancelFlightInfoRequest` later.

But I didn't have a plan to add `RenewFlightEndpointResult`. Should we add it?

NOTE: The ML message: https://lists.apache.org/thread/ofw6bydhyhcqqn3t8013gx0k6fbw8lvv

> * Remove CloseFlightInfo (if nobody objects it)
> * RefreshFlightEndpoint ->
>    RenewFlightEndpoint
> * RenewFlightEndpoint(FlightEndpoint) ->
>   RenewFlightEndpoint(RenewFlightEndpointRequest)
> * CancelFlightInfo(FlightInfo) ->
>   CancelFlightInfo(CancelFlightInfoRequest)

Reply:
I think your plan on the ML is fine!

Reply:
OK!